index.html

```diff
@@ -7,7 +7,12 @@
 	<meta name="designer" content="imgix">
 	<meta name="developer" content="imgix">
 	<meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,maximum-scale=1">
+	<!-- If CSP style policies, ensure SHA, nonce, or unsafe-inline are enabled -->
 	<meta http-equiv="Accept-CH" content="DPR, Width, Viewport-Width">
+	    <meta
+      http-equiv="Content-Security-Policy"
+      content="style-src 'self' 'sha256-W+PadZMOYbQi3EzR/NC+KcPx3bNLCz0OtfWwp+LK9b4=' 'sha256-OdI56ZW769BKGaLghLr3uZVaaWfqyRwyccMiu8gXf0w=' 'sha256-W+PadZMOYbQi3EzR/NC+KcPx3bNLCz0OtfWwp+LK9b4=' 'sha256-cG/+JJX3ZZs6snnMipxMnaYWATXBcyQw3vFWJF1zmAM='"
+    />
```

Comment:
Unfortunately this is the only way I successfully got around the CSP violation issues. We had to generate a SHA value for the style tags that violated the policy and add them to the csp. We also had to add `self` as an src to avoid the library's inline styles from getting blocked as well.

This will have to be done by an end user as well. Ie, they'll have to implement some combination of the [recommended](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src) options for `style-src` that allows our script and or styles to be set.

We could host the stylesheets in question on a CDN, but that seems like overkill when the user can do the above.